octoprint_CameraSettings/static/js/CameraSettings.js

```diff
@@ -64,10 +64,10 @@ $(function() {
             repeat_sequence_header: { use: ko.observable(false), value: ko.observable(undefined) },
             backlight_compensation: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
             pan_absolute: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
-            pan_relative: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
+            pan_relative: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(-4480), max: ko.observable(4480), step: ko.observable(0) },
             pan_speed: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
             tilt_absolute: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
-            tilt_relative: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(0), max: ko.observable(100), step: ko.observable(1) },
+            tilt_relative: { use: ko.observable(false), value: ko.observable(undefined), min: ko.observable(-1920), max: ko.observable(1920), step: ko.observable(0) },
```

Comment:
Min, max and step should never be hard coded here.
They are values reported by the camera and different cameras have different values. If you've got a camera that is misbehaving by reporting bad values then a different approach would be needed, such as what I've proposed in #51